test/ffmpeg-qsv/encode/avc.py

```diff
@@ -241,6 +241,26 @@ def test(self, case, rcmode, bitrate, maxrate, qp, quality, profile):
     self.init(spec, case, rcmode, bitrate, maxrate, qp, quality, profile)
     self.encode()
 
+class tcbrc_idr(AVCEncoderTest):
+  def init(self, tspec, case, rcmode, bitrate, maxrate, qp, quality, profile):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      rcmode     = rcmode,
+      bitrate    = bitrate,
+      case       = case,
+      maxrate    = maxrate,
+      minrate    = bitrate,
+      profile    = profile,
+      qp         = qp,
```

Comment:
tcbrc is a BRC, currently it is low power BRC. it should not include QP.

Reply:
Ok, I have revised and resubmitted